browser/ui/sidebar/sidebar_browsertest.cc

```diff
@@ -617,13 +617,18 @@ class SidebarBrowserWithSplitViewTest
     }
 
     scoped_features_.InitWithFeatureStates(
-        {{::features::kBraveWebViewRoundedCorners, std::get<0>(GetParam())},
-         {tabs::features::kBraveSplitView, std::get<1>(GetParam())},
+        {{tabs::features::kBraveSplitView, std::get<1>(GetParam())},
          {::features::kSideBySide, std::get<2>(GetParam())}});
   }
 
   ~SidebarBrowserWithSplitViewTest() override = default;
 
+  void SetUpOnMainThread() override {
+    SidebarBrowserTest::SetUpOnMainThread();
+    browser()->profile()->GetPrefs()->SetBoolean(kWebViewRoundedCorners,
```

Comment:
Same as speedreader, this is not the right place to test this. The sidebar test should be focused on sidebar functionality, not generic BraveBrowserView functionality

Reply:
for follow-up also since it appears this was already being tested here